contracts/interfaces/IUniswapV2ERC20.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity =0.5.16;
+pragma solidity >=0.5.0;
```

Comment:
```suggestion
pragma solidity =0.5.16;
```

Reply:
ignoring other changes, thoughts on this to indicate that this interface is for internal consumption?